legacy/firmware/fsm_msg_ethereum.h

```diff
@@ -45,7 +44,7 @@ void fsm_msgEthereumGetPublicKey(const EthereumGetPublicKey *msg) {
   if (!coin) return;
 
   if (!fsm_ethereumCheckPath(msg->address_n_count, msg->address_n, true,
-                             CHAIN_ID_UNKNOWN)) {
+                             NULL)) {
```

Comment:
just drop this check completely for `GetPublicKey`, same as was done in core

Reply:
9556b8f18d0dab6fc84e20f31bad8ba707737562